projects/ng-polymorpheus/src/directives/outlet.ts

```diff
@@ -62,47 +73,34 @@ export class PolymorpheusOutletDirective<C> implements OnChanges, DoCheck {
             this.process(this.content, proxy);
         } else if (
             // tslint:disable-next-line:triple-equals
-            (context instanceof PolymorpheusContext && context.$implicit) != null
+            (context instanceof PolymorpheusContext && context.$implicit) !== null
```

Comment:
I think it was meant to be like that to cover `undefined`, there's a tslint disable above

Reply:
Why not `!!(context instanceof PolymorpheusContext && context.$implicit)`